index.html

```diff
@@ -17,11 +17,42 @@
 <body>
 
     <div class="container">
+      <nav class="navbar navbar-expand-lg navbar-light  justify-content-end">
+        <a class="navbar-brand" href="#">Portfolio</a>
+        <a class="nav-link" href="#">Contact</a>
+        <a class="nav-link" href="#">Blog</a>
+      </nav>
       <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+          <div class="jumbotron col-7 justify-content-center">
```

Comment:
`jumbotron` is typically used as the main call-to-action, and usually occupies the full width of the webpage.

I would leave `jumbotron` as a separate `div` container and have another `div` container with `col-7` to wrap your other content.